File: shell-qa-test/aux_files/DeleteView_ClassicMode.py
session.run_sql('drop view if exists py_view;');
session.run_sql("create view py_view as select first_name from actor where first_name like '%a%';");
session.get_schema('sakila').get_views();
session.drop_view('sakila','py_view');

session.run_sql("SELECT table_name FROM information_schema.views WHERE information_schema.views.table_schema LIKE '%py_view%';");

File: shell-qa-test/aux_files/DeleteView_ClassicMode.js
session.runSql('drop view if exists js_view;');
session.runSql("create view js_view as select first_name from actor where first_name like '%a%';");
session.getSchema('sakila').getViews();
session.dropView('sakila','js_view');

session.runSql("SELECT table_name FROM information_schema.views WHERE information_schema.views.table_schema LIKE '%js_view%';");